desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -17,6 +17,7 @@
 
 import logging
 import json
+from prettytable import PrettyTable
```

Comment:
Note: will break with Python2 (as it won't be there), for now simpler to try/except ImportError

Reply:
Done